src/main/java/org/trivetia/mace/Mace.java

```diff
@@ -0,0 +1,56 @@
+package org.trivetia.mace;
+
+import net.minecraftforge.fml.common.Loader;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventHandler;
+import net.minecraftforge.fml.common.SidedProxy;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLServerStartingEvent;
+
+import org.trivetia.mace.common.CommonProxy;
+import org.trivetia.mace.ae2.spatial.SpatialIO;
+import org.trivetia.mace.commands.CommandSpatialAdd;
+
+@Mod(modid = Mace.MODID, version = Mace.VERSION, name = Mace.NAME)
+public class Mace {
+    public static final String MODID = "mace";
+    public static final String VERSION = "0.1";
+    public static final String NAME = "Minecraft Advanced Compatibility Extension";
+    
+    @SidedProxy(clientSide = "org.trivetia.mace.client.ClientProxy", serverSide = "org.trivetia.mace.server.ServerProxy")
+    public static CommonProxy proxy;
+    
+    @EventHandler
+    void preInit( FMLPreInitializationEvent event)
+    {
+        proxy.preInit(event);
+    }
+    
+    @EventHandler
+    void init( FMLInitializationEvent event )
+    {
+        proxy.init(event);
+        if( Loader.isModLoaded("appliedenergistics2") )

```

Comment:
You could move this into the proxy or create a singelton for all AE2 compat additions